python/CMakeLists.txt

```diff
@@ -36,7 +36,7 @@ check_python_module("skbuild" "scikit-build")
 execute_process(
   COMMAND
     ${PYTHON_EXECUTABLE} -c "from __future__ import print_function; \
-      import os; import skbuild; \
+    import os; import sys; temp = sys.stdout; sys.stdout = open(os.devnull,\"w\"); import skbuild; sys.stdout = temp; \
```

Comment:
Thanks for this PR! Will you be able to add a comment above this line that explains why we are doing this? something like "importing os and skbuild, while suppressing warnings from skbuild"?